frontend/src/components/transactions/ActionsList.tsx

```diff
@@ -5,7 +5,7 @@ import ActionRow from "./ActionRow";
 import { ViewMode, DetalizationMode } from "./ActionRowBlock";
 
 export interface Props {
-  actions: T.Action[];
+  actions?: T.Action[];
```

Comment:
There is no point in using this component if you don't have Actions, isn't it? I suggest we revert this change and conditionally use this component from the parent component based on the `actions` existence.